pkg/pillar/nistate/linux_flow.go

```diff
@@ -476,7 +492,7 @@ func (lc *LinuxCollector) processCapturedPacket(
 			}
 			return lc.processDADProbe(niInfo, packet)
 		}
-		if dnslayer != nil {
+		if dnslayer != nil && niInfo.config.EnableFlowlog {
```

Comment:
Does this mean that if we enable flow logging we will not have DNS information for previous DNS lookups? I don't know for how long time an application would remember the result of a DNS lookup before redoing it, but would it be expensive to record the DNS resolution in case we later enable flow logs?

Reply:
Good point. I think it is not that expensive to keep pcap and DNS packet processing enabled.
I have made the changes.
But to avoid the list of DNS records growing indefinitely, I decided to GC records older than one day.

Reply:
OK